src/main/java/frc/robot/RobotManager.java

```diff
@@ -26,12 +26,11 @@ public class RobotManager extends LoggedRobot {
 
 	private int roborioCycles;
 
-	private Command autonomousCommand;
+	private final Robot robot;
 
-	private Robot robot;
+	private Command autonomousCommand;
 
-	@Override
-	public void robotInit() {
+	public RobotManager() {
```

Comment:
Make sure the dependencies for WPILib are updated to support this change in the constructor

Reply:
It is